alerta-covid/app/controllers/api/v1/areas_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Api::V1::AreasController < Api::V1::ApiController
+  before_action :set_area, only: %i[ show ]
+  skip_before_action :verify_authenticity_token
+
+  # GET /areas or /areas.json
+  def index
+    @areas = Area.all
+    render json: @areas, status: :ok
+  end
+
+  # GET /areas/1 or /areas/1.json
+  def show
+    render json: @areas, status: :ok
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_area
+      @area = Area.find(params[:id])
+    end
+end
```

Comment:
Would be nice to add some tests to ensure this is behaving as expected 👍🏽

Reply:
Thanks! the tests are in the following commits [ test spec part 1](https://github.com/BC-NOV-20-Proyectos/alerta-covid-web-team01/pull/13/commits/6820cd8317c2d624dd3193ed07be38dfbfd37151) [ test spec part2](https://github.com/BC-NOV-20-Proyectos/alerta-covid-web-team01/pull/13/commits/6a68bcb443badd8caea6a43616319dae0962bfc8) [ test spec part3](https://github.com/BC-NOV-20-Proyectos/alerta-covid-web-team01/pull/13/commits/526b3826aea051503f9306d3eb94acae37f92b8d)